lib/writers/git.go

```diff
@@ -265,6 +266,21 @@ func (g *GitWriter) push(repo *git.Repository, logger logr.Logger) error {
 }
 
 func (g *GitWriter) cloneRepo(localRepoFilePath string, logger logr.Logger) (*git.Repository, error) {
+	// Azure DevOps requires multi_ack and multi_ack_detailed capabilities, which go-git doesn't
+	// implement. But: it's possible to do a full clone by saying it's _not_ _un_supported, in which
+	// case the library happily functions so long as it doesn't _actually_ get a multi_ack packet. See
+	// https://github.com/go-git/go-git/blob/v5.5.1/_examples/azure_devops/main.go.
+	oldUnsupportedCaps := transport.UnsupportedCapabilities
+
+	// This check is crude, but avoids having another dependency to parse the git URL.
+	if strings.Contains(g.GitServer.URL, "dev.azure.com") {
```

Comment:
not that familiar with azure devops, but do we know if this could be customisable by the tenant? like "dev.myorg.com" or something like that?

Reply:
a quick google makes me think not